src/plugins/cordova-plugin-network-information/sim-host.js

```diff
@@ -30,6 +37,18 @@ function initialize() {
     connectionTypeList.addEventListener('change', function () {
         network.updateNetworkType(connectionTypeList.value);
     });
+
+    registerTelemetryEvents();
+}
+
+function registerTelemetryEvents() {
+    var connectionList = document.getElementById('connection-list');
+
+    connectionList.onchange = function () {
+        var option = connectionList.options[connectionList.selectedIndex];
+
+        telemetry.sendUITelemetry(Object.assign({}, baseProps, { control: 'connection-list', value: option.value }));
+    };
```

Comment:
Now we are sending telemetry events for all of the usages of the simulation plugin. But what are we measuring and what would be the action to perform when we notice the usages of a plugin? Does it makes sense to measure all? What is the main goal of measuring?